src/api.ts

```diff
@@ -188,6 +188,7 @@ export class GrafanaApiClient implements GrafanaApi {
       {
         name: alert.name,
         state: alert.state,
+        matchingSelector: "",
```

Comment:
This is not really used by us anymore. But might be interesting to return the `dashboardTag`. Anyways, this is not really an important thing to be honest

Reply:
Changed that :) The naming is a bit off then because we're returning the tag and naming it `selector`. I guess it's fine?

Reply:
Yeah, it's totally fine. Since version 8 of Grafana the default is the unifiedAlerting, so this client will be not supported at some point, it's just to keep a bit of consistency in case we make those changes upstream (there are a bunch of PR still open for months :upside_down_face: )